collector/fixtures/sys.ttar

```diff
@@ -253,11 +253,6 @@ Lines: 1
 0
 Mode: 644
 # ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-Path: sys/class/infiniband/mlx4_0/ports/1/counters/port_xmit_discards
-Lines: 1
-0
-Mode: 644
-# ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
```

Comment:
Why is this fixture being removed, is it not used for anything?

Reply:
In the first version (reading directly the /sys filesystem) I regenerated the fixtures from my pc, maybe they were not present then... Since I switched to using prometheus API, I reset the file to the original state.